wcfsetup/install/files/lib/system/condition/UserMultifactorCondition.class.php

```diff
@@ -0,0 +1,141 @@
+<?php
+namespace wcf\system\condition;
+use wcf\data\condition\Condition;
+use wcf\data\DatabaseObjectList;
+use wcf\data\user\User;
+use wcf\data\user\UserList;
+use wcf\system\exception\InvalidObjectArgument;
+use wcf\system\exception\UserInputException;
+use wcf\system\WCF;
+
+/**
+ * Condition implementation for the multi-factor status of users.
+ * 
+ * @author	Joshua Ruesweg
+ * @copyright	2001-2020 WoltLab GmbH
+ * @license	GNU Lesser General Public License <http://opensource.org/licenses/lgpl-license.php>
+ * @package	WoltLabSuite\Core\System\Condition
+ * @since       5.4
+ */
+class UserMultifactorCondition extends AbstractSingleFieldCondition implements IContentCondition, IObjectListCondition, IUserCondition {
+	use TObjectListUserCondition;
+	
+	/**
+	 * @inheritDoc
+	 */
+	protected $label = 'wcf.user.condition.multifactor';
+	
+	/**
+	 * 1 if uses multifactor checkbox is checked
+	 * @var	integer
+	 */
+	protected $activeMultifactor = 0;
+	
+	/**
+	 * 1 if uses no multifactor checkbox is checked
+	 * @var	integer
+	 */
+	protected $inactiveMultifactor = 0;
```

Comment:
```suggestion
	protected $multifactorActive = 0;
	
	protected $multifactorNotActive = 0;
```

a) The comments are tautological.
b) Adjusted the wording to match the database column